crypto/pipe.go

```diff
@@ -7,27 +7,24 @@ import (
 	"sync"
 
 	"github.com/mergermarket/go-pkcs7"
-	// logging "github.com/ipfs/go-log"
 )
 
-// var log = logging.Logger("fula:crypto")
-
-const CHUNK_SIZE = 1024 * aes.BlockSize
+const ChunkSize = 1024 * aes.BlockSize
 
 type encoder struct {
 	reader   io.Reader
 	EnCipher Cipher
 }
 
 func NewEncoder(reader io.Reader) *encoder {
-	encipher, _ := NewEnCipher()
-	return &encoder{reader: reader, EnCipher: encipher}
+	encipher := NewEnCipher()
```

Comment:
isn't NewEnCipher returning multiple values? doesn;t it return an error if don't capture both?

Reply:
I modified this **NewEnCiper()** constructor. It now **returns only one argument.** Because during the generation of the constructor there is no place where this error could occur.